recipes/zulu-openjdk/all/test_package/conanfile.py

```diff
@@ -1,16 +1,27 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 from six import StringIO
```

Comment:
We don't support python 2 anymore.
```suggestion
from io import StringIO
```

Reply:
thanks, fixed!